Lazydocs: API Reference doc refresh (#1424)
wandb/docs: content/en/guides/integrations/paddleocr.md

    use_wandb: True
```

> Any additional, optional arguments that you might like to pass to [`wandb.init`]({{< relref "/ref/python/sdk/functions/init" >}}) can also be added under the `wandb` header in the yaml file:

Reviewer: ```suggestion
You can instead pass additional optional arguments to [`wandb.init`]({{< relref "/ref/python/sdk/functions/init" >}}) in the `wandb` key in the YAML file:
```
